storio-common/src/main/java/com/pushtorefresh/storio/operations/PreparedOperation.java

```diff
@@ -37,4 +38,14 @@
     @NonNull
     @CheckResult
     Observable<Result> createObservable();
+
+    /**
+     * Creates {@link rx.Single} that emits result of Operation.
```

Comment:
Can you please change to `that emits result of Operation lazily when somebody subscribes to it`